systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -180,10 +187,10 @@ public static String updatePolicyOfRealmClient(String namespaceName, String base
      * @return result of creation
      */
     public static String importRealm(String namespaceName, String baseURI, String token, String realmData) {
-        String coPodName = kubeClient(namespaceName).getClusterOperatorPodName();
+        final String testSuiteScraperPodName = ResourceManager.kubeClient().listPodsByPrefixInName(Constants.TEST_SUITE_NAMESPACE, Constants.SCRAPER_NAME).get(0).getMetadata().getName();
```

Comment:
wouldn't be better to use some label and list the pod based on it?

Reply:
I can do it :)